ci/build_setup.sh

```diff
@@ -114,8 +114,8 @@ mkdir -p "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/bazel
 mkdir -p "${ENVOY_CI_DIR}"/bazel
 ln -sf "${ENVOY_SRCDIR}"/bazel/get_workspace_status "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/bazel/
 ln -sf "${ENVOY_SRCDIR}"/bazel/get_workspace_status "${ENVOY_CI_DIR}"/bazel/
-ln -sf "${ENVOY_SRCDIR}"/.bazelrc "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/
-ln -sf "${ENVOY_SRCDIR}"/.bazelrc "${ENVOY_CI_DIR}"/
+cp "${ENVOY_SRCDIR}"/.bazelrc "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/
+cp "${ENVOY_SRCDIR}"/.bazelrc "${ENVOY_CI_DIR}"/
```

Comment:
What's the reason for this change? It leads to an error and early termination of CI step for me:
```
$ ENVOY_DOCKER_BUILD_DIR=~/build ./ci/run_envoy_docker.sh './ci/do_circle_ci.sh bazel.clang_tidy'
disk space at beginning of build:
Filesystem      Size  Used Avail Use% Mounted on
overlay         504G  322G  163G  67% /
tmpfs            64M     0   64M   0% /dev
tmpfs            97G     0   97G   0% /sys/fs/cgroup
/dev/sda1       504G  322G  163G  67% /build
shm              64M     0   64M   0% /dev/shm
tmpfs            97G     0   97G   0% /proc/acpi
tmpfs            97G     0   97G   0% /sys/firmware
No remote cache bucket is set, skipping setup remote cache.
ENVOY_SRCDIR=/source                                                 
HEAD is now at 3e5b733... cleanup: match NOT_IMPLEMENTED_GCOVR_EXCL_LINE change (#53)
cp: '/source/.bazelrc' and '/source/ci/.bazelrc' are the same file
disk space at end of build:
Filesystem      Size  Used Avail Use% Mounted on
overlay         504G  322G  163G  67% /
tmpfs            64M     0   64M   0% /dev
tmpfs            97G     0   97G   0% /sys/fs/cgroup
/dev/sda1       504G  322G  163G  67% /build
shm              64M     0   64M   0% /dev/shm
tmpfs            97G     0   97G   0% /proc/acpi
tmpfs            97G     0   97G   0% /sys/firmware
```

Reply:
i.e. you should either revert it back or change to `cp -f`.

Reply:
the bazel-compile-database doesn't read bazel options from env variable so the script dumps to bazelrc. this keeps git clean without writing to original file.